### PR TITLE
[cron-jobs/validation] Update & move to Quay.io

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Review by lbarcziova — APPROVED** [review]
thanks

**Comment by jpopelka** [comment]
recheck

File: cron-jobs/validation/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM fedora:32
+FROM fedora:33
 
 RUN dnf install -y python3-ogr python3-copr python3-pip && dnf clean all
 
-RUN pip install --upgrade sentry-sdk==0.14.2
+RUN pip3 install --upgrade sentry-sdk && pip3 check
```
Comment (TomasTomecek) [inline cron-jobs/validation/Dockerfile:5]: I'm assuming you need to update this change in the cluster manually, correct?

**Review by TomasTomecek — APPROVED** [review]
LGTM, would be awesome to roll this out before Friday so we have this functioning 100% on Monday

File: cron-jobs/validation/Dockerfile (hunk 1 above)
Comment (jpopelka) [reply cron-jobs/validation/Dockerfile:5]: are you talking about this line or the whole PR?
I'm assuming the latter :-)
Yes, I've just re-deployed the job.

**Pull request merged**